api/urls.py

```diff
@@ -55,20 +55,18 @@
         "v1/hunts/<int:hunt_id>/puzzles/<int:pk>", puzzle_detail, name="puzzle_detail"
     ),
     path(
-        "v1/hunts/<int:hunt_id>/puzzles/<int:puzzle_id>/answers",
+        "v1/puzzles/<int:puzzle_id>/answers",
```

Comment:
Can we make `v1` a variable and share it across `urls.py` and test files?

Reply:
I'm a little hesitant because I don't want it to ever vary (we should never change it--the only reason to bump the version would be to maintain backwards compatibility => the existing code & tests should continue using the same endpoints) and it feels like any constant is going to be less readable than just the two characters "v1". So I bias towards just leaving it until it's actually a problem